development/sp3-local/config/config.php

```diff
@@ -85,11 +120,11 @@
     'errorreporting' => false,
 
     /*
-     * Custom error show function called from \SimpleSAML\Error\Error::show.
+     * Custom error show function called from SimpleSAML_Error_Error::show.
```

Comment:
I copied the content of sp2-local/config/config.php into this one, thinking consistency was better than whatever might have been in the sildisco repo. Seeing this now, I see that I overlooked some details. But since it looks like this type of error is only in comments, I think I'll leave it. We have another ticket to make sure files like this are updated using new SSP templates.